RUFAS/biophysical/animal/animal_grouping_scenarios.py

```
@@ -36,6 +36,17 @@ class AnimalGroupingScenario(Enum):
         ]
     }
 
+    BEEF_COW_CALF_HERD = {
+        AnimalCombination.BEEF_COW_CALF_PAIR: [AnimalType.BEEF_COW, AnimalType.BEEF_CALF],
+        AnimalCombination.BEEF_GESTATING: [AnimalType.BEEF_COW],
+        AnimalCombination.BEEF_REPLACEMENT: [AnimalType.BEEF_HEIFER_REPLACEMENT],
+        AnimalCombination.BEEF_BULL_BATTERY: [AnimalType.BEEF_BULL],
+    }
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)